### ISSUE TITLE
Consider using YAML to capture run metadata in run_nmx etc.

### ISSUE THREAD

**Issue by MikeKSmith** [RichardHooijmaijers/shinyMixR#2]
The run information metadata which is currently defined within a function might fit a YAML header format nicely, which could open up (re)use with markdown documents as the header information of a report.

**Comment by RichardHooijmaijers** [comment]
I agree that yaml would fit nicely for the meta data. This has implications beyond ShinyMixR and will discuss this in the team to investigate feasibility for nlmixr as well
